scripts/protopackage.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/sh
+set -eo pipefail
+
+# This script appends the "option go_package" proto option to the file located at $FNAME.
+# This option is not included in the proto files to allow the files to more easily
+# be hosted in github.com/tendermint/spec and shared between other repos.
+# This option specifies what the package will be named when imported by other packages.
+
+FNAME=$1
+MODNAME=$(echo $2| sed 's/\//\\\//g')
+PACKAGE="$(dirname $FNAME | sed 's/^\.\/\(.*\)/\1/g' | sed 's/\//\\\//g')"
+
+if ! `grep -q 'option\s\+go_package\s\+=\s\+.*;' $FNAME`; then 
```

Comment:
Do we need the backticks here? I don't think we want the output, just the exit status, right? (Ideally the output will be empty anyway)
```suggestion
if ! grep -q 'option\s\+go_package\s\+=\s\+.*;' "$FNAME" ; then 
```

Reply:
Also: I think for the `\s` and such to work, we will need to use `grep -E`, as POSIX grep assumes "basic" regexp without that flag. (GNU may do clever things, not sure)

Reply:
Don't we need to replace the `go_package` even if the file already defines one?

Reply:
Ah, this script worked on my machine, so I'm guessing that the GNU version behaves differently. Added -E

Reply:
Hmm, I ran this on my mac without -E and it seemed to work, mind double checking me? oddly, with -E, it fails?

Reply:
Hmm. Do you maybe have GNU grep installed via homebrew or something?